src/test/py/bazel/bzlmod/bazel_vendor_test.py

```diff
@@ -502,7 +497,7 @@ def testBuildingOutOfDateVendoredRepo(self):
     # external and not a symlink
     self.assertIn(
         "WARNING: <builtin>: Vendored repository '+ext+justRepo' is"
-        ' out-of-date. The up-to-date version will be fetched into the external'
+        ' out-of-date (repo rule changed). The up-to-date version will be fetched into the external'
```

Comment:
I guess this will also include cases where the starlark semantic is changed due to some flag?

Reply:
That sounds pretty enough for me, thanks!